util.go

```diff
@@ -34,3 +36,19 @@ func getRemoteURLGitLab(org, repo string) (string, error) {
 	}
 	return remoteURL, nil
 }
+
+func getGitHubOrganization(webhookType string, webhookEvent interface{}) (string, error) {
+	switch webhookType {
+	case "pull_request":
+		pr := webhookEvent.(*github.PullRequestEvent)
+		return *pr.Organization.Login, nil
+	case "push":
+		push := webhookEvent.(*github.PushEvent)
+		return *push.Repo.Organization, nil
+	case "issue_comment":
+		comment := webhookEvent.(*github.IssueCommentEvent)
+		return *comment.Repo.Owner.Login, nil
```

Comment:
FYI, we should be able to get `Organization.Login` from all kind of events. I opened a PR upstream to have these objects available. See https://github.com/google/go-github/pull/2116